src/NumericalAlgorithms/DiscontinuousGalerkin/BoundarySchemes/FirstOrder/FirstOrderSchemeLts.hpp

```diff
@@ -118,13 +115,13 @@ struct FirstOrderSchemeLts {
       const gsl::not_null<db::item_type<::Tags::Mortars<mortar_data_tag, Dim>>*>
           all_mortar_data,
       const Mesh<Dim>& volume_mesh,
-      const db::const_item_type<
-          ::Tags::Mortars<domain::Tags::Mesh<Dim - 1>, Dim>>& mortar_meshes,
-      const db::const_item_type<
-          ::Tags::Mortars<::Tags::MortarSize<Dim - 1>, Dim>>& mortar_sizes,
+      const typename ::Tags::Mortars<domain::Tags::Mesh<Dim - 1>, Dim>::type&
+          mortar_meshes,
+      const typename ::Tags::Mortars<::Tags::MortarSize<Dim - 1>, Dim>::type&
+          mortar_sizes,
       const NumericalFlux& normal_dot_numerical_flux_computer,
-      const db::const_item_type<time_stepper_tag>& time_stepper,
-      const db::const_item_type<Tags::TimeStep>& time_step) noexcept {
+      const typename time_stepper_tag::type::element_type& time_stepper,
```

Comment:
Here we assume the time stepper is stored in the DataBox as a unique_ptr. That's the case because it can be selected at runtime from the input file. Replacing `db::const_item_type` with `Tag::type::element_type` makes this code depend on the choice how we store the tag in the DataBox and if we make it runtime-selectable or not. It seems to me that reduces maintainability of the code. For example, if we made something like the analytic solution or the EOS selectable at runtime, we would have to go through the entire code and replace `Tag::type`s with `Tag::type::element_type`. Using `db::const_item_type` makes the code agnostic to that choice. What do you think?